src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -2003,9 +1996,41 @@ var create_overlay = function (app) {
                 error_url_after_new_tab: true,
                 error_message: null,
                 lightbox: null,
+                prevActiveElement: null,
             }
         },
+        props: {
+            frame_src: String,
+        },
         methods: {
+        },
+        watch: {
+            frame_src: function (newValue) {
+                this.$el.querySelector("iframe").src = newValue;
+            },
+            frame_loading: function (newValue) {
+                if (newValue) {
+                    this.$el.querySelector("iframe").src = this.frame_src;
+                }
```

Comment:
Can you explain the idea behind this, maybe even in a comment?

Doesn't that possibly cause
```
        this.$root.overlay.frame_loading = true;
        this.$root.overlay.frame_src = redirect_url;
```
to trigger two page loads, one to the old and one to the new URL?

Reply:
The problem I faced, was that frame_src was not changed when I re-opened an existing cart. This always made the loading-spinner stick. Originally the iframe.src was always set and that happened to trigger a load-event. To mimick this behaviour, I actively set the iframe.src on frame_loading. Probably not the most elegant and definitely not the proper solution. I will look into this more. Sorry!